source/documentors/references/templates/doc_concept_template.rst

```diff
@@ -0,0 +1,66 @@
+Concept Document Template
+#########################
+
+A concept is a document that clarifies and illuminates a particular topic. It is understanding-oriented. It could be considered a conceptual guide. This type of document answers a why question. It could be descriptive, historical, or even propose different alternatives to explain the bigger picture and give context. 
+
+For example, “What is an XBlock? The Open edX platform provides different components, called XBlocks, that can work to create a course, like text, video, assessment, and discussions.”  (Replace this text with your content.)
+
+Since this type of document seeks to explain a topic, you will likely need to add images to exemplify the topic you are talking about. For example:
+
+
+.. image:: /_images/documentors_howto/image_example.png
```

Comment:
Why is the example text above about XBlocks, but this image unrelated to XBlocks? It's confusing. I'd make the image and the example text above at line 6 match.